modules/homepage/src/main/frontend/src/themePage/index.jsx

```diff
@@ -0,0 +1,125 @@
+//  under the License.
+//
+import PropTypes from 'prop-types';
+import React from "react";
+import ReactDOM from "react-dom";
+import Sidebar from "./Sidebar/sidebar"
+import sidebarRoutes from './routes';
+import { withStyles } from '@material-ui/core/styles';
+import { Redirect, Router, Route, Switch } from "react-router-dom";
+import { createBrowserHistory } from "history";
+import Navbar from "./Navbars/Navbar";
+import IndexStyle from "./indexStyle.jsx";
+
+// Generate the switch/routes for our router to render components
+const switchRoutes = (
+  <Switch>
+    {sidebarRoutes.map((prop, key) => {
+      return (
+        <Route
+          path={prop.layout + prop.path}
+          component={prop.component}
+          key={key}
+        />
+      );
+    })}
+  </Switch>
+);
+
+class Main extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      image: "/libs/lfs/resources/cancer-cells.jpg",
+      color: "blue",
+      hasImage: true,
+      fixedClasses: "dropdown show",
+      mobileOpen: false
+    };
+  }
+
+  handleDrawerToggle = () => {
+    this.setState({ mobileOpen: !this.state.mobileOpen });
+  };
+
+  // Close the mobile menu if the window size changes
+  // so that the mobile menu is out of place
+  autoCloseMobileMenus = event => {
+    if (window.innerWidth >= this.props.theme.breakpoints.values.md) {
+      this.setState({ mobileOpen: false });
+    }
+  }
+
+  // Register/unregister autoCloseMobileMenus to window resizing
+  componentDidMount() {
+    window.addEventListener("resize", this.autoCloseMobileMenus);
+  };
+  componentWillUnmount() {
+    window.removeEventListener("resize", this.autoCloseMobileMenus);
+  };
+
+  render() {
+    const { classes, ...rest } = this.props;
+
+    return (
+      <div className={classes.wrapper}>
+        <Sidebar
+          routes={sidebarRoutes}
+          logoText={"LFS Data Core"}
+          logoImage={"/libs/lfs/resources/lfs-logo-tmp-cyan.png"}
+          image={this.state.image}
+          handleDrawerToggle={this.handleDrawerToggle}
+          open={this.state.mobileOpen}
+          color={"blue"}
+          {...rest}
+        />
+        <div className={classes.mainPanel} ref="mainPanel">
+          <Navbar
+            routes={sidebarRoutes}
+            handleDrawerToggle={this.handleDrawerToggle}
+            {...rest}
+          />
+          <div className={classes.content}>
+            <div className={classes.container}>{switchRoutes}</div>
+          </div>
+          <div id="footer-container"></div>
+        </div>
+      </div>
+    );
+  }
+}
+
+Main.propTypes = {
+  classes: PropTypes.object.isRequired
+};
+const MainComponent = (withStyles(IndexStyle, {withTheme: true})(Main));
+
+const hist = createBrowserHistory();
+ReactDOM.render(
+  <Router history={hist}>
+    <Switch>
+      <Route path="/" component={MainComponent} />
+      <Redirect from="/" to="/content/dashboard.html" />
```

Comment:
Do you mean this instead?

```suggestion
      <Redirect from="/" to="/content.html/dashboard.html" />
```

Reply:
For some reason, I can't redirect from "/content.html" to anywhere, meaning that visiting /content.html leads you to a blank page with the sidebar.